AntennaTracker/GCS_MAVLink_Tracker.cpp

```diff
@@ -63,6 +63,87 @@ MAV_STATE GCS_MAVLINK_Tracker::vehicle_system_status() const
     return MAV_STATE_ACTIVE;
 }
 
+void GCS_MAVLINK_Tracker::send_attitude_target()
+{
+    Quaternion quat;
+    bool use_yaw_rate = false;
+    float yaw_rate_rads = 0.0;
+    bool use_pitch_rate = false;
+    float pitch_rate_rads = 0.0;
+    float quat_out[4] {0.0, 0.0, 0.0, 0.0};
+    uint16_t typemask = ATTITUDE_TARGET_TYPEMASK_BODY_ROLL_RATE_IGNORE |
+                        ATTITUDE_TARGET_TYPEMASK_THROTTLE_IGNORE;
+
+    if (tracker.mode->number() == Mode::Number::GUIDED) {
```

Comment:
thanks for this, normally we wouldn't check the mode number like this.  Instead we normally add methods onto the mode classes.